lnrpc/lightning.proto

```diff
@@ -772,6 +772,18 @@ message GetTransactionsRequest {
 message TransactionDetails {
     // The list of transactions relevant to the wallet.
     repeated Transaction transactions = 1;
+
+    /*
+    The index of the last item in the set of returned transactions. This can be
+    used to seek further, pagination style.
+    */
+    uint64 last_index_offset = 2;
```

Comment:
This should be zero-based and correspond to the slice index used to slice into transactions.
Also, a better name would probably be just `last_index` and `first_index`, as they aren't necessarily always influenced by the input `offset` field.